src/AWS/Signers/V4.elm

```diff
@@ -44,6 +44,7 @@ algorithm =
 headers : AWS.Config.Service -> List ( String, String )
 headers config =
     [ ( "Host", AWS.Http.host config.endpoint )
+    , ( "Accept", "application/json" )
```

Comment:
As far as I know yes. Force a json response so that we can use our decoders. Otherwise we get xml. Tested with query protocol